src/util/permission.js

```diff
@@ -21,11 +21,15 @@ const hasPermission = (userId, permission) => {
                 attributes: ['type', 'permission'],
             }],
         }).then((result) => {
-            resolve(
-                JSON.parse(
-                    result.role.permission
-                )[permission.realm].includes(permission.action)
-            );
+            if (result == null) {
+                reject('Cannot find role for the given id');
+            } else {
```

Comment:
Cant we simplify this into one line like 
```JS
if (result == null) return reject('Cannot find role for the given id');
resolve()
```

Reply:
@singhabahu if i do this it shows a maximum length lint issue. So i think better to keep as it is.